### PR TITLE
Updating subprocess.pyi to support python 3.5

### PR DESCRIPTION
**Pull request by wreed4**
Also added DEVNULL which was added in 3.3

### CODE REVIEW

**Comment by wreed4** [comment]
This is in response to #437

File: stdlib/3/subprocess.pyi [hunk 1]
```diff
@@ -3,7 +3,42 @@
 # Based on http://docs.python.org/3.2/library/subprocess.html
 
 import sys
-from typing import Sequence, Any, Mapping, Callable, Tuple, IO, Optional, Union
+from typing import Sequence, Any, Mapping, Callable, Tuple, IO, Optional, Union, List
+
+
+if sys.version_info >= 3.5:
```
Comment (matthiaskramm) [inline stdlib/3/subprocess.pyi]: (3, 5)

File: stdlib/3/subprocess.pyi [hunk 2]
```diff
@@ -3,7 +3,42 @@
...
+    class CompletedProcess:
+        args = ...  # type: Union[List, str]
+        returncode = ... # type: int
+        stdout = ... # type: Union[str, bytes]
+        stderr = ... # type: Union[str, bytes]
+        def __init__(self, args: Union[List, str], 
+                     returncode: int, 
+                     stdout: Union[str, bytes], 
+                     stderr: Union[str, bytes]): ...
```
Comment (matthiaskramm) [inline stdlib/3/subprocess.pyi]: -> None

**Comment by wreed4** [comment]
@matthiaskramm When should this be available for use in mypy?  I don't know how this repository is released and such.

**Comment by matthiaskramm** [comment]
I'm told they update their typeshed submodule once a week or so. So pretty soon I guess?

**Comment by gvanrossum** [comment]
Yeah, watch for a commit to the mypy repo saying "Sync typeshed."

--Guido (mobile)

**Pull request merged**